test/message/message.status

```diff
@@ -16,3 +16,6 @@ prefix message
 
 [$system==freebsd]
 
+[$system==aix]
```

Comment:
https://github.com/nodejs/node/pull/29054, this also was removed on master

Reply:
Cherry-picked cleanly